src/internal/pfsdb/branches.go

```diff
@@ -731,6 +748,21 @@ func DeleteBranchTrigger(ctx context.Context, tx *pachsql.Tx, from BranchID) err
 	return nil
 }
 
+func GetBranchPropagationSpecDirectSubvenanceRows(ctx context.Context, ext sqlx.ExtContext, id BranchID) ([]*BranchPropagationSpecRow, error) {
```

Comment:
I think you can probably omit 'DirectSubvenanceRows' from the name of this function.

Reply:
The name is a mouthful, but I think it is important that it is clear we are getting the direct subvenance rather than full subvenance or provenance.